lib/internal/Magento/Framework/App/FeedFormatsInterface.php

```diff
@@ -0,0 +1,11 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Framework\App;
+
+interface FeedFormatsInterface
+{
+    const DEFAULT_FORMAT = 'rss';
```

Comment:
I don't think that it's very flexible to have  the dedicated interface for holding one constant only.

Reply:
@ldusan84 yes, Agree.

So, what I propose is to add constant from `FeedsFormatsInterface` into ` \Magento\Framework\App\FeedFactoryInterface`

and constants from `FeedOutputFormatsInterface` into ` Magento\Framework\App\FeedInterface`

Because only these entities are dependent on these constants.
Doing that we would not introduce a lot of inextensible interfaces and keep following SRP